src/hatti/views/record.cljs

```diff
@@ -32,7 +32,7 @@
         new-rank ((case dir :left dec :right inc) cur-rank)]
     [:a {:on-click (click-fn
                     #(put! shared/event-chan {:submission-to-rank new-rank}))
-         :class "pure-button" :href "#"} [:i {:class icon}]]))
+         :class "pure-button btn-default" :href "#"} [:i {:class icon}]]))
```

Comment:
Same comment as from Zebra. Perhaps we should just make the styles bound to `.btn-default` appear _by default_ on all buttons and then override them? Then we wouldn't need to have a btn-default class for people to remember to add when working on the UI.